### ISSUE TITLE
prison warden

### ISSUE THREAD

**Issue by GoogleCodeExporter** [alistairreilly/andors-trail#101]
```
I just tried to complete the quest with the prison and I defeated the warden 
but he didn't drop a necklace. Just wondering if this is a defect or just not 
completed in this version. I am using version 6.7 on an LG Ally.


```

Original issue reported on code.google.com by `Captain8...@gmail.com` on 21 Dec 2010 at 7:05

**Comment by GoogleCodeExporter** [comment]
```
worked A-OK on 0.6.7b2, Android 2.1. Haven't tried twice, though.
```

Original comment by `surrano` on 21 Dec 2010 at 7:17

**Comment by GoogleCodeExporter** [comment]
```
I did defeat the warden before checking out the demon though. I found out about 
the missing necklace by talking to the sentry outside. I double checked my 
inventory and the warden only dropped a bone, a potion and a ruby. Another 
reason may be that when i fought him, at the last second another creature 
appeared and so i had to fight both at the same time. I killer the warden first 
and then the loot screen came up before i had killed the other creature. I 
thought that was pretty strange too. I'm just using the straight 6.7 that came 
out 22 hours ago.
```

Original comment by `Captain8...@gmail.com` on 21 Dec 2010 at 7:25

**Comment by GoogleCodeExporter** [comment]
```
So I just went back and checked my save game files and found that i had saved 
right before fighting the warden and i defeated him again, and this time the 
necklace showed up. That means it must have been the other creature popping up.
```

Original comment by `Captain8...@gmail.com` on 21 Dec 2010 at 7:28

**Comment by GoogleCodeExporter** [comment]
```
Perhaps it was still on the floor?
```

Original comment by `SamuelPl...@gmail.com` on 22 Dec 2010 at 10:47

**Comment by GoogleCodeExporter** [comment]
```
Is this an unresolved issue or should we close this?
```

Original comment by `oskar.wi...@gmail.com` on 9 Jan 2011 at 6:31

**Comment by GoogleCodeExporter** [comment]
```
I think this is mentioned in another issue: if you fight a boss + wandering 
monster but kill boss first then don't get boss treasure... or other way 
around, can't recall
```

Original comment by `surrano` on 9 Jan 2011 at 3:05

**Comment by GoogleCodeExporter** [comment]
```
I think i posted this issue first. It happens when you kill the Warden first 
and then the wandering monster. I wound up going back to an older save point 
and continuing. When I fought the warden again alone, i received the necklace. 
I wouldn't say the issue is resolved, but a very rare occurrence.
```

Original comment by `Captain8...@gmail.com` on 9 Jan 2011 at 5:50

**Comment by GoogleCodeExporter** [comment]
```
There might have been a bug here. See discussion in issue 56 and issue 108.

I'm marking this issue as "Ready" since the fix in issue 56 should have solved 
this now.

Thank you for reporting it.
```

Original comment by `oskar.wi...@gmail.com` on 13 Jan 2011 at 9:55
- Changed state: **Ready**
- Added labels: **Milestone-0.6.8**

**Comment by GoogleCodeExporter** [comment]
```
where does this quest chain start? 
when i go to the prison, i'm only able to talk to the warden about the prisoner.
```

Original comment by `c.nennew...@googlemail.com` on 22 Jan 2011 at 12:15

**Comment by GoogleCodeExporter** [comment]
```
@c.nennew: this is not about Fallhaven prison. There's another one in the 
wilderness, and not for the faint-hearted ;)
```

Original comment by `surrano` on 24 Jan 2011 at 8:27

**Comment by GoogleCodeExporter** [comment]
```
Does status "Ready" mean this fix has been applied? I ran into this problem 
yesterday. Definitely have the latest version. Had to restart the game. Was 
very frustrating.
```

Original comment by `steve.st...@googlemail.com` on 7 Feb 2011 at 11:24

**Comment by GoogleCodeExporter** [comment]
```
Yes it does, but you forgot to read the label which is used to indicate the 
version - namely v0.6.8 as in the next version.
```

Original comment by `kims...@gmail.com` on 7 Feb 2011 at 12:47

**Comment by GoogleCodeExporter** [comment]
```
Thanks I noticed that just after posting the question
```

Original comment by `steve.st...@googlemail.com` on 8 Feb 2011 at 10:07

**Comment by GoogleCodeExporter** [comment]
```
Released in v0.6.8 . Closing.
```

Original comment by `oskar.wi...@gmail.com` on 24 Feb 2011 at 9:34
- Changed state: **Fixed**
